Cows-HacknSlash/Assets/Scripts/Inventory script/Additem.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Additem : MonoBehaviour {
+
+    public Inventory invenscript;
```

Comment:
Personal opinion: Maybe it's nicer to rename this to either `inv_script` or since you're using an Inventory object, just calling it `public Inventory inventory` or `public Inventory inv`